addon/templates/users/edit.hbs

```diff
@@ -73,7 +73,7 @@
               name="phone"
               placeholder="{{t "emeis.users.headings.phone-placeholder"}}..."
               required={{includes "phone" this.requiredFields}}
-              pattern="^[0-9()\-\s]+$"
+              pattern="^[0-9()\+\-\s]+$"
```

Comment:
The `+` should probably only be allowed at the beginning tho... but then again, I'm opting for being less restrictive in general - you can't really validate all possible cases anyway